<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>app</class>
 <widget class="QMainWindow" name="app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>app</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textboxInputFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>201</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textboxOutputFile">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>201</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReadInputFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerateGraph">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Graph</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGraph">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>791</width>
      <height>621</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtboxInputRegex">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Regular Expression:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReadRegex">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
